src/Swashbuckle.AspNetCore.SwaggerGen/SwaggerGenerator/SwaggerGenerator.cs

```diff
@@ -421,16 +421,18 @@ private OpenApiOperation GenerateOpenApiOperationFromMetadata(ApiDescription api
                 foreach (var content in requestContentTypes)
                 {
                     var requestParameters = apiDescription.ParameterDescriptions.Where(desc => desc.IsFromBody() || desc.IsFromForm());
-                    if (requestParameters.Any())
+                    var countOfParameters = requestParameters.Count();
+                    if (countOfParameters >= 1)
                     {
                         _ = requestParameters.SingleOrDefault(desc => desc.IsFromBody());
```

Comment:
What does this achieve given the result is discarded?

Reply:
It will throw when there is more than one Body document, as it was doing previously. It does not make sense to have more than 1 Body element

Reply:
Computing it as a side-effect to throw a not-very-helpful exception message doesn't seem to be desirable behaviour to me. If we want to stop this if it occurs, then we should explicitly check for it and then throw a useful exception, rather than a vague "Sequence contains more than one element" exception.

Reply:
Finally I tried several ways to introduce several FromBody (Passing AsParameters and then FromBody or putting FromBody in just one element and a exception is thrown before reaching Swashbucle code (A binding error)